fastlane/spec/fixtures/plist/Info.plist

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
As a side effect of #6917 or more accurately shenzhen#11 fastlane uses `plist-3.2.0`. Now that the Shenzhen dependency is gone, this is still true. So every time I run the tests locally, I've this local modification. It actually should have been part of 41e840a, but since the change is related to Shenzhen, I think it should be part of this PR.